solanches/controller.py

```diff
@@ -70,6 +70,15 @@ def adiciona_destaques(destaques, comercio_nome):
     Comercio.add_destaques(comercio_nome, filtered_destaques)
 
 
+def remove_produto(comercio_nome, produto_id):
+    comercio = Comercio.get_by_name(comercio_nome)
+    assert comercio, f'Erro: comercio com nome {comercio_nome} nao cadastrado!'
+    produtos_comercio = Comercio.get_produtos(comercio_nome)
+    assert produto_id in produtos_comercio, f'Erro: produto precisa fazer parte do cardápio do comércio'
+    
+    Comercio.delete_produto(comercio_nome, produto_id)
```

Comment:
É bom manter a consistência nos nomes dos métodos
```suggestion
    Comercio.remove_produto(comercio_nome, produto_id)
```